src/core/geometry/path/create.js

```diff
@@ -0,0 +1,19 @@
+const mat4 = require('../../math/mat4')
+
+/**
+ * Produces an empty, open path.
+ * @returns {path2} - the empty, open path.
+ * @example
+ * create()
+ */
+const create = () => {
+  return {
+    basePoints: [],  // Contains canonical, untransformed points.
```

Comment:
ok so I will be honest, I do not like have two copies of points at all and fail to see the point (haha)
- double memory cost 
- canonicalization (which, again, I have my reservation about, I still thing you over evaluate the accumulated errors, but that is another debate) should always happen : or put another way:
   * if your `isCanonicalized` is set to true, your points are canonical, if not they simply are not, and you do not need to double the storage

Reply:
Double a small amount is a small amount.

The choice is between performing transforms upon transformed data incrementally, or performing a single accumulated transform upon the original data.

Reply:
@kaosat-dev The plan is to keep the lazy transforms now, and do some profiling later across the library when things start to function again. If there are performance or memory issues the lazy transforms can be removed.